README.md

```diff
@@ -0,0 +1,50 @@
+# Portofilo project
+
+This is the a portofilo project  
+
+![screenshot]('./Screenshot from 2021-10-21 03-58-06.png')
```

Comment:
Good job using the template provided by Microverse to your readme 🙂 

- [ ] Please fix the link to this image, web browsers don't read very well spaces in name files... with something simpler like `![screenshot]('./screenshot')` should do. Remember to change the name of the file too so they match.